<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CVToolsClass</class>
 <widget class="QMainWindow" name="CVToolsClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>888</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CVTools</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Rounded</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>891</width>
      <height>591</height>
     </rect>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Rounded</enum>
    </property>
    <property name="currentIndex">
     <number>3</number>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideNone</enum>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="maskingTab">
     <attribute name="icon">
      <iconset resource="qtcvtools.qrc">
       <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emotes/22/face-angel.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emotes/22/face-angel.svg</iconset>
     </attribute>
     <attribute name="title">
      <string>Masking Tool</string>
     </attribute>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>30</y>
        <width>551</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="maskViewButton">
         <property name="text">
          <string>Mask View</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="normalViewButton">
         <property name="text">
          <string>Normal View</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hsvViewButton">
         <property name="text">
          <string>HSV View</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>670</x>
        <y>40</y>
        <width>181</width>
        <height>501</height>
       </rect>
      </property>
      <property name="title">
       <string>Values</string>
      </property>
      <widget class="QSlider" name="vmaxSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>450</y>
         <width>158</width>
         <height>18</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="hmaxHeaderLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>96</y>
         <width>158</width>
         <height>46</height>
        </rect>
       </property>
       <property name="text">
        <string>Hmax:</string>
       </property>
      </widget>
      <widget class="QLabel" name="vminHeaderLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>323</y>
         <width>158</width>
         <height>45</height>
        </rect>
       </property>
       <property name="text">
        <string>Vmin:</string>
       </property>
      </widget>
      <widget class="QSlider" name="hmaxSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>148</y>
         <width>158</width>
         <height>18</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="vmaxHeaderLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>398</y>
         <width>158</width>
         <height>46</height>
        </rect>
       </property>
       <property name="text">
        <string>Vmax:</string>
       </property>
      </widget>
      <widget class="QSlider" name="sminSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>223</y>
         <width>158</width>
         <height>18</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="smaxHeaderLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>247</y>
         <width>158</width>
         <height>46</height>
        </rect>
       </property>
       <property name="text">
        <string>Smax:</string>
       </property>
      </widget>
      <widget class="QLabel" name="hminHeaderLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>21</y>
         <width>158</width>
         <height>45</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>Hmin:</string>
       </property>
      </widget>
      <widget class="QLabel" name="sminHeaderLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>172</y>
         <width>158</width>
         <height>45</height>
        </rect>
       </property>
       <property name="text">
        <string>Smin:</string>
       </property>
      </widget>
      <widget class="QSlider" name="hminSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>72</y>
         <width>158</width>
         <height>18</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="smaxSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>299</y>
         <width>158</width>
         <height>18</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QSlider" name="vminSlider">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>374</y>
         <width>158</width>
         <height>18</height>
        </rect>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="hminValue">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>20</y>
         <width>41</width>
         <height>45</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="hmaxValue">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>100</y>
         <width>41</width>
         <height>45</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="sminValue">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>170</y>
         <width>41</width>
         <height>45</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="smaxValue">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>250</y>
         <width>41</width>
         <height>45</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="vminValue">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>320</y>
         <width>41</width>
         <height>45</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
      <widget class="QLabel" name="vmaxValue">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>400</y>
         <width>41</width>
         <height>45</height>
        </rect>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="midLineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>10</y>
        <width>411</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Picture Screen</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QToolButton" name="openToolBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>40</width>
        <height>40</height>
       </rect>
      </property>
      <property name="contextMenuPolicy">
       <enum>Qt::PreventContextMenu</enum>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qtcvtools.qrc">
        <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/apps/16/system-file-manager.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/apps/16/system-file-manager.svg</iconset>
      </property>
      <property name="autoRaise">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>20</width>
        <height>521</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QScrollArea" name="scrollArea">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>100</y>
        <width>551</width>
        <height>441</height>
       </rect>
      </property>
      <property name="widgetResizable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="scrollAreaWidgetContents">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>549</width>
         <height>439</height>
        </rect>
       </property>
       <widget class="QLabel" name="imageLabel">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>551</width>
          <height>451</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="scanningTab">
     <attribute name="icon">
      <iconset resource="qtcvtools.qrc">
       <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/actions/16/node-transform.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/actions/16/node-transform.svg</iconset>
     </attribute>
     <attribute name="title">
      <string>Scanning Tool</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Scanning Screen</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>650</x>
        <y>10</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Output Field</string>
      </property>
     </widget>
     <widget class="QLabel" name="outputImage_scn">
      <property name="geometry">
       <rect>
        <x>510</x>
        <y>40</y>
        <width>351</width>
        <height>501</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="openToolBox_scan">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qtcvtools.qrc">
        <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/apps/16/system-file-manager.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/apps/16/system-file-manager.svg</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="addPointer_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtcvtools.qrc">
        <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/vcs-added.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/vcs-added.svg</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="deletePointer_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>170</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtcvtools.qrc">
        <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/vcs-removed.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/vcs-removed.svg</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="scanPermit_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>230</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtcvtools.qrc">
        <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/emblem-checked.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/emblem-checked.svg</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="saveOutput_btn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>290</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtcvtools.qrc">
        <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/actions/16/document-export.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/actions/16/document-export.svg</iconset>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="modelTrainerTab">
     <attribute name="icon">
      <iconset resource="qtcvtools.qrc">
       <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/categories/32/applications-development.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/categories/32/applications-development.svg</iconset>
     </attribute>
     <attribute name="title">
      <string>Model Trainer</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="objectDetectorTab">
     <attribute name="icon">
      <iconset resource="qtcvtools.qrc">
       <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/categories/32/applications-engineering.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/categories/32/applications-engineering.svg</iconset>
     </attribute>
     <attribute name="title">
      <string>Object Detector</string>
     </attribute>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>256</width>
        <height>491</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="objDetector_output">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>581</width>
        <height>541</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::Box</enum>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QToolButton" name="removeCascade_btn">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtcvtools.qrc">
        <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/vcs-removed.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/vcs-removed.svg</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="openToolBox_objectDetector">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="qtcvtools.qrc">
        <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/apps/16/system-file-manager.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/apps/16/system-file-manager.svg</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="addCascade_btn">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtcvtools.qrc">
        <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/vcs-added.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/vcs-added.svg</iconset>
      </property>
     </widget>
     <widget class="QToolButton" name="startDetection_btn">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>10</y>
        <width>41</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="icon">
       <iconset resource="qtcvtools.qrc">
        <normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/emblem-checked.svg</normaloff>:/QTDeneme/C:/Users/emir-/OneDrive/Masa端st端/breeze-icons-master/icons/emblems/16/emblem-checked.svg</iconset>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>888</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFiles">
    <property name="title">
     <string>Files</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout_Us"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionZoom_In"/>
    <addaction name="actionZoomOut"/>
    <addaction name="actionNormal_Size"/>
    <addaction name="separator"/>
    <addaction name="actionFit_the_Window"/>
   </widget>
   <addaction name="menuFiles"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <action name="actionAbout_Us">
   <property name="text">
    <string>About Us</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="text">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoomOut">
   <property name="text">
    <string>ZoomOut</string>
   </property>
  </action>
  <action name="actionNormal_Size">
   <property name="text">
    <string>Normal Size</string>
   </property>
  </action>
  <action name="actionFit_the_Window">
   <property name="text">
    <string>Fit the Window</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtcvtools.qrc"/>
 </resources>
 <connections/>
</ui>
